crates/router/src/connector/adyen/transformers.rs

```diff
@@ -1651,7 +1652,11 @@ fn get_recurring_processing_model(
         (_, Some(true)) => Ok((
             Some(AdyenRecurringModel::UnscheduledCardOnFile),
             None,
-            Some(format!("{}_{}", item.merchant_id, item.get_customer_id()?)),
+            Some(format!(
+                "{}_{}",
+                item.merchant_id,
+                item.get_customer_id()?.get_string_repr()
+            )),
```

Comment:
Why do we need a string representation, can we use domain type instead

Reply:
We need to get the string representation here because we are formatting the `customer_id` using underscore.